addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -2432,11 +2432,11 @@ var BasicModel = AbstractModel.extend({
                     const modelFieldName = record.fields[modelField].string;
                     const referenceFieldName = record.fields[fieldName].string;
 
-                    this.do_warn(_t(`'${referenceFieldName}' is unsynchronized
+                    this.displayNotification({ title: _t(`'${referenceFieldName}' is unsynchronized
                                      with '${modelFieldName}'.`),
```

Comment:
The translation seens completely broken here. Even if babel can extract the source string, `_t` is going to receive the string post-interpolation.

Reply:
I guess the issue is already present before the PR then? I'll double check and suggest fix in the appropriate version.

Reply:
Yes, I just saw that because the PR touches it but it should be broken regardless.

Reply:
I think we were talking about forbidding the ```_t(`...)``` with @d-fence

Reply:
Just ``` _t(`...`) ``` can be useful for example if there are both `"` and `'` in the string it's convenient to have another delimiter to avoid having to escape anything. It's the use of `${}` inside it that is an issue I suppose?

Reply:
Yes, there will be a linter. Already working in that [branch](https://github.com/odoo-dev/odoo/tree/master-lint-template-strings-moc). No PR yet

Reply:
This was changed, I rebased on it.